tutorial/Lab1/README.md

```diff
@@ -0,0 +1,168 @@
+# Lab 1. I just want to deploy!
+
+We developers are a lazy bunch! If we can get away with just typing a single command over multiple, not worrying about how things tie together, then we are more than happy. Lets investigate how Helm can help with our lack of work ethic by first deploying an application to a Kubernetes cluster using `kubectl` and afterwards using Helm.
```

Comment:
`Lets` should be `Let's`

Reply:
It is funny (about lack of work ethic), but perhaps this is better:
```Let's investigate how Helm can help us focus on other things by letting a chart do the work for us. We'll first deploy an application to a Kubernetes cluster by using `kubectl` and then show how we can offload the work to a chart by deploying the same app with Helm. ```

Reply:
I think you have written it more eloquently. Noted and changed. Thanks.